alembic/versions/20250626_bc471d8a83fb_remove_superseded_concept_from_license_pools.py

```diff
@@ -0,0 +1,33 @@
+"""Remove superseded concept from license pools.
+
+Revision ID: bc471d8a83fb
+Revises: 87051f7b2905
+Create Date: 2025-06-26 20:24:52.724565+00:00
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "bc471d8a83fb"
+down_revision = "87051f7b2905"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.drop_column("licensepools", "superceded")
+
+
+def downgrade() -> None:
+    op.add_column(
+        "licensepools",
+        sa.Column(
+            "superceded",
+            sa.BOOLEAN(),
+            autoincrement=False,
+            nullable=False,
+            server_default=sa.false(),
```

Comment:
@tdilauro, I believe the migration tests are failing because `server_default` is being treated differently from `default`. Since `server_default` is a database-side configuration and isn't set in the existing code, the migration is detecting a discrepancy in the database configuration.

It seems you're using `server_default` to bypass the nullable constraint. To ensure consistency in the configuration, you can either:  
1. Create the column as nullable, populate it, and then set it to non-nullable, or  
2. Create the column with `server_default` and later modify it to drop the `server_default`.

Reply:
Thanks, @jonathangreen!